db/index.js

```diff
@@ -28,4 +28,4 @@ const productDetailsSchema = new mongoose.Schema({
 
 const productDetails = mongoose.model('productDetails', productDetailsSchema);
 
-module.exports.productDetails = productDetails;
\ No newline at end of file
+module.exports.productDetails = productDetails;

```

Comment:
There might be a more intuitive way to export this https://nodejs.org/api/modules.html